src/olympia/addons/tasks.py

```diff
@@ -697,7 +698,13 @@ def migrate_legacy_dictionaries_to_webextension(ids, **kw):
         ids[0], ids[-1], len(ids))
     addons = list(Addon.objects.filter(id__in=ids))
     for addon in addons:
-        migrate_legacy_dictionary_to_webextension(addon)
+        try:
+            migrate_legacy_dictionary_to_webextension(addon)
+        except ValidationError as exp:
```

Comment:
We are primarily using `as exc` instead of `as exp` except in src/olympia/api/tests/test_authentication.py and `extract_strict_compatibility_value_for_addon` in `addons/tasks.py` :man_shrugging: